components/Tabs2/tests/snapshot/MDCTabBarViewItemViewSnapshotTests.m

```diff
@@ -54,4 +51,17 @@ - (void)testItemViewNoTitleNoImageIntrinsicContentSize {
   [self generateSnapshotAndVerifyForView:itemView];
 }
 
+- (void)testItemViewLongTitleRegularImageIntrinsicContentSize {
```

Comment:
Looks like this snapshot test golden image didn't get added. If you're not sure how to add the goldens, let's sync-up tomorrow in GVC and I'll walk you through it.

Reply:
I think the golden image is added now.